### DISCUSSION TITLE
Remarks as a user :)

### DISCUSSION THREAD

**Discussion by aTrotier** [MagneticResonanceImaging/MRIReco.jl#62] [General]
Hi,
I will use that topic to make some remarks of stuff that feels weird to me.

# coherence in dimensions 
Most of the time the dimensions of an array are `(sx,sy,sz,echos,channel)`

Because I'm a BART user It is already weird for me ^^, but something less intuitive is that the dimensions for the sensitivity maps has to be `(sx,sy,sz,channel)` in the reconstruction function

I suppose one of the reasons is : sens[:,:,:,1,:] remove the echo dimension

**Comment by aTrotier** [comment]
The compressed sensing example (with TV) in the doc is not working in 3D. 
The proxTV operator only works for 2D, no ?

**Reply by tknopp** [reply]
Probably. Jakob already generalized quite a bit of code in RegularizedLeastSquares to ND.
